website/client/src/components/payments/sendGemsModal.vue

```diff
@@ -294,6 +294,7 @@ export default {
       this.sendingInProgress = false;
     },
     close () {
+      this.$root.$emit('habitica::dismiss-modal', 'send-gems');
```

Comment:
could we have the `this.$root.$emit('bv::hide::modal', modalName);` call inside the `habitica::dismiss-modal` event handler?

Reply:
Hmm, I think when I tried that it caused some issues, but I'll try again and document what happened--maybe you'll know how to fix it!

Reply:
Ahh, I think that must have been before I implemented the bit that removes the `prev` data. So it does work, and that makes it much easier to use--we can just call `habitica::dismiss-modal` anywhere we want instead of `bv::hide::modal`!